### ISSUE TITLE
Incorrect statusPageUrl and healthCheckUrl are reported to Eureka (1.3.1.RELEASE)

### ISSUE THREAD

**Issue by Aloren** [spring-cloud/spring-cloud-netflix#2144]
Currently in our application we have setup the following properties:
```
server.contextPath=/MyService
server.port=8097
management.port=8297
```
In previous versions of spring-cloud-netflix we had the following statusPageUrl and healthCheckUrl reported to Eureka:
```
<statusPageUrl>
http://host:8297/info
</statusPageUrl>
<healthCheckUrl>
http://host:8297/health
</healthCheckUrl>
```
I've just updated version of spring-cloud-netflix to 1.3.1.RELEASE and got:
```
<statusPageUrl>
http://host:8297/MyService/info
</statusPageUrl>
<healthCheckUrl>
http://host:8297/MyService/health
</healthCheckUrl>
```
As you can see from the diff, context path was added, new links are not valid.

I've found the following [ticket](https://github.com/spring-cloud/spring-cloud-netflix/issues/1795).
It looks to me that the following line:
```
String managementContextPath = propertyResolver.getProperty("management.contextPath", propertyResolver.getProperty("server.contextPath", "/"));
``` 
is not correct. Since `server.contextPath` is not applied to management endpoints.

**Comment by Aloren** [comment]
I've created pull request: https://github.com/spring-cloud/spring-cloud-netflix/pull/2145

**Comment by spencergibb** [comment]
Closed by #2145
